### PR TITLE
Updates to code of conduct & license

### PR DESCRIPTION
**Pull request by bridgetkromhout**
Updates to code of conduct & license

Signed-off-by: Bridget Kromhout <bridget@kromhout.org>

### CODE REVIEW

**Comment by bridgetkromhout** [comment]
Seems it's still going to circleci, without a config?

**Comment by bridgetkromhout** [comment]
Ah, CircleCI is still set here: https://github.com/servicemeshinterface/smi-metrics/settings/hooks - on a repo-specific basis. @lachie83 should we remove that, do you think?

**Comment by Pothulapati** [comment]
@bridgetkromhout I think, we should go ahead and remove this repo hooks with Circle CI, as we [just moved to Github Actions](https://github.com/servicemeshinterface/smi-metrics/pull/58), 

We don't see GH Actions Checks in the PR because they only run for code changes as per https://github.com/servicemeshinterface/smi-metrics/blob/master/.github/workflows/main.yml#L8 :)

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
+# Community
```
Comment (Pothulapati) [inline README.md]: Hmm, Usually `community` is at the end of the readme right?

**Comment by bridgetkromhout** [comment]
>  Okay, that hook was successfully deleted.

I removed the repo-specific CircleCI webhook.

**Pull request merged**